VEHICLE/IsVehicleStoppedAtTrafficLights.md

```diff
@@ -9,10 +9,11 @@ BOOL IS_VEHICLE_STOPPED_AT_TRAFFIC_LIGHTS(Vehicle vehicle);
 ```
 
 ```
-Is this for red lights only?  more testing required.
+This native checks if the given vehicle is stopped at a red light (this won't return true if the vehicle is stopped behind another vehicle waiting at a traffic light)
```

Comment:
Change for:

Check if given vehicle is stopped at a red light. 

**Note:** will return `false` if the vehicle is stopped behind another vehicle waiting at a traffic light.

In my opinion.